src/library_dylink.js

```diff
@@ -802,7 +802,7 @@ var LibraryDylink = {
           if (cSig != 'void') {
             cSig = cSig.split(',');
             for (var i in cSig) {
-              var jsArg = cSig[i].split(' ').pop();
+              var jsArg = cSig[i].split(' ').pop().replace('*', '');
```

Comment:
Doesn't this `replace` already happen on the line below?

Reply:
Ah I see you are trying to replace more than one asterisk.    Can you instead change the line below to use `replaceAll`?